.github/workflows/delivery.yml

```diff
@@ -1,11 +1,16 @@
 name: Delivery
```

Comment:
Maybe rename workflow (+ filename?) to something more suitable, since it now doesn't necessarily deliver anything (on schedule and PR)

Reply:
I can address this in a separate PR